tests/test_interpreter.py

```diff
@@ -2,32 +2,33 @@
 from random import randint
 import time
 import pytest
-import interpreter
+import interpreter as i
```

Comment:
I think we should give this a more useful and readable name than `i`. Maybe using PascalCase and calling it `Interpreter` would be better?